.github/workflows/docker.yml

```
@@ -0,0 +1,25 @@
+name: Docker Build and Push
+
+on:
+  push:
+    tags:
+      - "*"
+

```

Concurrent group would be nice2have.
Because with the current configuration the last action that finishes will be marked at the latest instead of the latest published tag.
```suggestion

concurrency:
  group: "docker-inage"
  cancel-in-progress: false
```

---

Added! Thanks for the heads-up